examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Why not iterate directly over the individual p rows instead of over the indexes?</p><p> </p><pre class="ql-syntax" spellcheck="false">y = np.stack([sp.stats.multinomial(n=total_count, p=p_vector).rvs()[0] for p_vector in p])
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>or</p><pre class="ql-syntax" spellcheck="false">np.vstack([sp.stats.multinomial(n=total_count, p=p_vector).rvs() for p_vector in p])
</pre><p>so there's no need for the mysterious <code>[0]</code>?</p>

Reply:
<blockquote>Why not iterate directly over the individual p rows instead of over the indexes?</blockquote><p> </p><p>Yep, good call, @ricardoV94. Definitely more pythonic.</p><p> </p><pre class="ql-syntax" spellcheck="false">np.vstack([sp.stats.multinomial(n=total_count, p=p_vector).rvs() for p_vector in p])
</pre><p> </p><p>Ooh, I like that, @MarcoGorelli. Will use.</p>